Scripts/test-CocoaPods-integration.sh

```diff
@@ -1,37 +1,51 @@
 #!/bin/bash
 
+# Any subsequent(*) commands which fail will cause the shell script to exit immediately
+set -eo pipefail
+
 function print_help {
   echo "Test CocoaPods Integration"
   echo " "
   echo "test-CocoaPods-integration [-w]"
   echo " "
   echo "options:"
   echo "-w, --exclude-wechat      exclude wechat module"
```

Comment:
this should be `--include-wechat` not `--exclude-wechat`, I think I made a mistake :-)